ui/jobs/buff_tracker.ts

```diff
@@ -104,6 +104,12 @@ export class Buff {
     const readyKey = 'r:' + this.name + ':' + source;
     this.ready[source] = this.makeAura(readyKey, this.readyList, -1, 0,
         this.readySortKeyBase, color, txt, 0.6);
+
+    // if a readied raidbuff not be used in 3min, we can assume that
+    // this player has left the battlefield, or at least his raidbuff is unexpectable.
+    setTimeout(() => {
+      this.ready[source]?.removeCallback();
+    }, 3 * 60 * 1000);
```

Comment:
I think some raid buffs have 3 minute cooldowns (and sometimes you hold that three minute cooldown for a bit to line up with other buffs).  So, I think either the time needs to be extended to like 5-6 minutes or possibly some other way to approach this could be better?

One option might be to only clear cooldowns for zones that are not raids and trials? (There's a ZoneInfo/ContentType to tell you this).  Another option is to not show cooldowns at all in bozja.  Do either of these options sound better?

Reply:
If you hold a **readied** 3 minutes raidbuff for 3 minutes, you will absolutely lose at least one time of possible use, and I think one should never do it. I think 3 minutes is enough. I've not remembered a battle can have downtime more than 3 minutes, even so you will know that everything is ready like opener.
There's also situation that one member abondon duty in raids and another join party in progress, so I think zoneinfo is not a good choice.

Reply:
Oh, "readied".  Hmm!  That's true.  Only really potions at that point start being something you could hold for 3 minutes and still be worth it.

I guess I was trying to differentiate if you felt like this was an issue in raiding content or only in bozja/eureka/open world.  It sounds like you think this would be helpful everywhere though.  That sounds good!